CMakeLists.txt

```diff
@@ -74,6 +113,28 @@ target_include_directories(${LIB_NAME}
     ${OPENSSL_INCLUDE_DIR}
 )
 
+###
+###  Library packaging and install
+###
+#include ( CPack )
+set ( dist_dir    ${CMAKE_BINARY_DIR}/dist )
+set ( prefix      ${CMAKE_INSTALL_PREFIX} )
+set ( exec_prefix ${CMAKE_INSTALL_PREFIX}/bin )
+set ( libdir      ${CMAKE_INSTALL_PREFIX}/lib )
```

Comment:
I would prefer `lib_dir`, as with `dist_dir` above.  In any case, be consistent about whether there's an underscore.